packages/agent/conf/demos/dbc/presentation_definitions/SphereonWalletIdentityCredential.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "dbc2023",
```

Comment:
Since we are asking for a SphereonWalletIdentityCredential, shouldn't it be "sphereonWalletIdentity"?
If so, then also rename dbcConferenceAttendee.json to SphereonWalletIdentityCredential.json in oid4vp_options and change dbc2023 to sphereonWalletIdentity
It's dbcConferenceAttendee we are issueing

Reply:
changed to `sphereonWalletIdentity` and also changed related places